commons/src/test/java/org/sonarsource/analyzer/commons/RuleMetadataLoaderTest.java

```diff
@@ -530,20 +531,43 @@ class TestRule {
     RulesDefinition.Rule rule = context.repository(RULE_REPOSITORY_KEY).rule("S2092");
     assertThat(rule.type()).isEqualTo(RuleType.SECURITY_HOTSPOT);
     assertThat(rule.securityStandards())
-      .containsExactlyInAnyOrder("cwe:311", "cwe:315", "cwe:614", "owaspTop10:a2", "owaspTop10:a3", "owaspTop10-2021:a4", "owaspTop10-2021:a5");
+      .containsExactlyInAnyOrder("cwe:311", "cwe:315", "cwe:614",
+        "owaspTop10:a2", "owaspTop10:a3",
+        "owaspTop10-2021:a4", "owaspTop10-2021:a5");
```

Comment:
Nitpick: Since this is a list, I prefer not to have two values on the same line, as it suggests a key-value mapping.